### DISCUSSION TITLE
[Question]: spurious gpt-3.5 requests

### DISCUSSION THREAD

**Discussion by clarity99** [danny-avila/LibreChat#2944] [Q&A] [answered]
### What is your question?

I am seeing some strange behaviour in LibreChat - everytime I ask for example gpt-4o a question, I also get 2 or 1 gpt-3.5. calls

### More Details

in openai platform activity for the key I use only for librechat I see 1 call for gpt-4o and 2 for gpt-35-turbo. What is LibreChat doing with 35turbo calls?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Title generation. You should only see an extra call at the start of a conversation

**Comment by clarity99** [comment]
thank you!
